app/routes/articles.js

```diff
@@ -24,23 +24,27 @@ router.post('/', isLoggedIn, hasAuthorRole, (req, res) => {
         return res.redirect('/')
     }
 
-    const {title, description, body} = req.body
-    const author = {id: req.user._id, username: req.user.username}
+    const link = encodeURIComponent(req.body.title.replace(SPACES, DASH))
+    const title = encodeString(req.body.title)
```

Comment:
Maybe call it sanitize, there's so much encoding and string replacing that it starts to be hard to tell things apart.

Reply:
Or convertToHtmlEntities